controllers/apply/form-router-next/lib/file-uploads.js

```diff
@@ -19,6 +20,22 @@ const s3 = new AWS.S3({
     region: 'eu-west-2'
 });
 
+const ClamScan = new NodeClam().init({
+    debug_mode: true,
+    scan_recursively: false,
+    clamdscan: {
+        socket: '/usr/local/etc/clamd.socket',
```

Comment:
We'll want any paths to come from secrets or the environment so we can determine where to look depending on the environment (e.g. EC2 or locally)

Reply:
Good shout! I will move these to the environment files as these are env related details.